src/pages/XeniumAnalyser.tsx

```diff
@@ -47,6 +49,7 @@ type LabwareSamples = {
 export type XeniumAnalyserFormValues = {
   lotNumberA: string;
   lotNumberB: string;
+  equipmentId: number | undefined;
```

Comment:
Just to note, if you want to make it in a more ts way  you can always use equipmentId?: number which is the same as number|undefined